packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -168,8 +168,16 @@ export const _FileDownloadCards = (props: _FileDownloadCards): JSX.Element => {
   );
 
   const fileDownloadHandler = useCallback(
-    async (userId, file) => {
+    async (userId, file: FileMetadata) => {
       if (!props.downloadHandler) {
+        /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+        if (file.previewUrl) {
```

Comment:
don't think this is a good idea though, might be more reliable to check based on file type ("teamsFile" vs. "fileSharing")?

Reply:
In UI Component it has lost the information 'teamsFile' as it will convert 'file' to 'fileSharing' in the binding layer. The FileMetadataAttachmentType is only 'fileSharing', 'teamsInlineImage', and 'unknown'

Reply:
resolving this to continue the same conversion in the other comment i left